pkg/controller/node/nodecontroller.go

```diff
@@ -746,6 +748,38 @@ func (nc *NodeController) deletePods(nodeName string) (bool, error) {
 	return remaining, nil
 }
 
+// update ready status of all pods running on given node from master
+// return true if success
+func (nc *NodeController) updatePods(nodeName string, readyStatus api.ConditionStatus) (bool, error) {
```

Comment:
given that this is called on true-> false but not false->true, maybe this should be less generically named and parameterized?  setPodsNotReady(node) or something?